data/docs/frontend-monitoring/sending-logs.mdx

```diff
@@ -1,8 +1,9 @@
 ---
 date: 2025-08-17
-tags : [SigNoz Cloud, Self-Host]
+tags: [SigNoz Cloud, Self-Host, Frontend, JavaScript]
 title: Sending Logs from your frontend application
 id: sending-logs-frontend
+description: Send logs from your frontend application to SigNoz
```

Comment:
Update to `Learn how to send frontend logs to SigNoz using OpenTelemetry for complete visibility across your application.`